java 1.8でjarが作れない

以下ログ

```
$ java -version
java version "1.8.0_151"
Java(TM) SE Runtime Environment (build 1.8.0_151-b12)
Java HotSpot(TM) 64-Bit Server VM (build 25.151-b12, mixed mode)

$ ./gradlew assemble
> Task :compileJava FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':compileJava'.
> Could not target platform: 'Java SE 11' using tool chain: 'JDK 8 (1.8)'.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

Deprecated Gradle features were used in this build, making it incompatible with Gradle 6.0.
Use '--warning-mode all' to show the individual deprecation warnings.
See https://docs.gradle.org/5.1.1/userguide/command_line_interface.html#sec:command_line_warnings

BUILD FAILED in 0s
1 actionable task: 1 executed

--- a/build.gradle
+++ b/build.gradle
@@ -20,8 +20,8 @@ apply plugin: 'jacoco'
 apply plugin: 'maven'
 
 // Set compiler version
-sourceCompatibility = '1.11'
-targetCompatibility = '1.11'
+//sourceCompatibility = '1.11'
+//targetCompatibility = '1.11'
 
 // Set default encoding
 compileJava.options.encoding = 'UTF-8'

